modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -212,9 +211,8 @@ protected void finished(Description description) {
         this.stop();
     }
 
-    private void stopAndRetainRecording(Description description) {
-        File recordingFile = new File(vncRecordingDirectory, "recording-" + filenameDateFormat.format(new Date()) + ".flv");
-
+    private void stopAndRetainRecording(Description description, boolean succeeded) {
```

Comment:
Is there a way to rename this method such that the meaning of the parameter becomes more clear in the using code?
Maybe something along the lines of  `stopAndRetainRecordingForDescriptionAndSuccessState` (very long, yes =/).

Reply:
Sure. Need smalltalk / Objective-C / scala named args :-)